index.html

```diff
@@ -1,7 +1,7 @@
 <title>snuggsi.esツ</title>
 
 <meta name=viewport content=width=device-width,initial-scale=1>
-<meta name="theme-color>
+<meta name=theme-color>
```

Comment:
Where's the `content=` property @albertoponti? i think that's the value `theme-color` is setting. Should be some hexadecimal color value like #bada55